### ISSUE TITLE
The examples pending_message is always `Not yet implemented` ? 

### ISSUE THREAD

**Issue by sawanoboly** [rspec/rspec-core#2633]
### Subject of the issue

I am trying to create a custom formatter. Since the pending_message included in the example which is passed to fommater is different from the pending argument, I would like to use argument of pending for it.

### Your environment
* Ruby version:

ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-darwin18]

* rspec-core version:

RSpec 3.8
  - rspec-core 3.8.0
  - rspec-expectations 3.8.3
  - rspec-mocks 3.8.0
  - rspec-support 3.8.0

### Steps to reproduce

here is example spec.

```ruby
describe 'foo' do
  pending 'bar'
end
```

result is below

```
$ rspec -f j ./example_spec.rb | jq .
{
  "version": "3.8.0",
  "examples": [
    {
      "id": "./example_spec.rb[1:1]",
      "description": "bar",
      "full_description": "foo bar",
      "status": "pending",
      "file_path": "./example_spec.rb",
      "line_number": 2,
      "run_time": 8e-06,
      "pending_message": "Not yet implemented"
    }
  ],
  "summary": {
    "duration": 0.000741,
    "example_count": 1,
    "failure_count": 0,
    "pending_count": 1,
    "errors_outside_of_examples_count": 0
  },
  "summary_line": "1 example, 0 failures, 1 pending"
}
```

### Expected behavior

Isn't the argument of pending supposed to be used for pending_message?

```
{
      "id": "./example_spec.rb[1:1]",
      "description": "foo",
      "full_description": "foo",
      "status": "pending",
      "file_path": "./example_spec.rb",
      "line_number": 2,
      "run_time": 8e-06,
      "pending_message": "bar"
    }
```

### Actual behavior

Same as the Steps to reproduce example. The pending argument is treated the same as a normal describe.

**Comment by benoittgt** [comment]
If you look at "message" meaning in RSpec source code you will see it refers to very different subject.

See: https://github.com/rspec/rspec-core/blob/58f38210492cd369784d3fe1a849c0e81342a2f2/lib/rspec/core/example_group.rb#L147

Why not using "description" argument?

For me "pending_message" is more an internal RSpec sentence.

See
```ruby
# ruby this_file.rb

# frozen_string_literal: true

begin
  require "bundler/inline"
rescue LoadError => e
  $stderr.puts "Bundler version 1.10 or later is required. Please update your Bundler"
  raise e
end

gemfile(true) do
  source "https://rubygems.org"

  gem "rspec"
end

RSpec.configure do |config|
  config.formatter = :json
end

puts "Ruby version is: #{RUBY_VERSION}"
require 'rspec/autorun'

RSpec.describe 'pending' do
  it "skip because it fails after" do
    pending("something else getting finished")
    fail
  end

  it "raise because nothing after" do
    pending("nothing after")
  end

  describe 'foo' do
    pending 'bar'
  end
end
```

```json
{
   "version":"3.8.0",
   "examples":[
      {
         "id":"rspec_pending.rb[1:1]",
         "description":"skip because it fails after",
         "full_description":"pending skip because it fails after",
         "status":"pending",
         "file_path":"rspec_pending.rb",
         "line_number":24,
         "run_time":0.000101,
         "pending_message":"something else getting finished"
      },
      {
         "id":"rspec_pending.rb[1:2]",
         "description":"raise because nothing after",
         "full_description":"pending raise because nothing after",
         "status":"failed",
         "file_path":"rspec_pending.rb",
         "line_number":29,
         "run_time":5.5e-05,
         "pending_message":"nothing after",
         "exception":{
            "class":"RSpec::Core::Pending::PendingExampleFixedError",
            "message":"Expected example to fail since it is pending, but it passed.",
            "backtrace":[
               "rspec_pending.rb:29"
            ]
         }
      },
      {
         "id":"rspec_pending.rb[1:3:1]",
         "description":"bar",
         "full_description":"pending foo bar",
         "status":"pending",
         "file_path":"rspec_pending.rb",
         "line_number":34,
         "run_time":4.0e-06,
         "pending_message":"Not yet implemented"
      }
   ],
   "summary":{
      "duration":0.001087,
      "example_count":3,
      "failure_count":1,
      "pending_count":2,
      "errors_outside_of_examples_count":0
   },
   "summary_line":"3 examples, 1 failure, 2 pending"
}
```

in documentation output
```
pending
  skip because it fails after (PENDING: something else getting finished)
  raise because nothing after (FAILED - 1)
  foo
    bar (PENDING: Not yet implemented)

Pending: (Failures listed here are expected and do not affect your suite's status)

  1) pending skip because it fails after
     # something else getting finished
     Failure/Error: fail
     RuntimeError:
     # rspec_pending.rb:26:in `block (2 levels) in <main>'

  2) pending foo bar
     # Not yet implemented
     # rspec_pending.rb:34


Failures:

  1) pending raise because nothing after FIXED
     Expected pending 'nothing after' to fail. No error was raised.
     # rspec_pending.rb:29

Finished in 0.00267 seconds (files took 0.23614 seconds to load)
3 examples, 1 failure, 2 pending

Failed examples:

rspec rspec_pending.rb:29 # pending raise because nothing after
```

**Comment by JonRowe** [comment]
TL;DR; The problem is that`pending` in a describe is an empty example. So its not implementing. If you give it a block it will expect it to fail. Skip metadata will provide a message.
